static.json

```diff
@@ -2,5 +2,6 @@
   "root": "build/",
   "routes": {
     "/**": "index.html"
-  }
+  },
+  "https_only": true
```

Comment:
Hopefully this will work to redirect & pass the lighthouse PWA audit once this is built on Heroku